pkg/cluster/internal/loadbalancer/const.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package loadbalancer
 
 // Image defines the loadbalancer image:tag
-const Image = "kindest/haproxy:v20210715-a6da3463"
+const Image = "kindest/haproxy:latest"
```

Comment:
pinned versions please, otherwise we can't detect problems caused by different versions

Reply:
ahh, got it